packages/zowe-explorer/README.md

```diff
@@ -358,6 +358,8 @@ For more information about SCS, see [Secure Credential Store Plug-in for Zowe Ex
 
 - **Associate profiles**: You can create a secondary association by right-clicking the profile and selecting the **Associate profiles** icon. For more information, see [the Associate profiles section](https://docs.zowe.org/stable/user-guide/ze-profiles.html#associate-profile) in Zowe Docs.
```

Comment:
@IgorCATech This is a bit nitpicky and not technically part of the PR, but could we change "**Associate profiles** icon" to "**Associate profiles** option"?

Reply:
This is actually a good catch. "Icon" here is an obsolete term since the function is no longer initiated by clicking the icon that used to be part of the UI. Now, this feature is activated from the drop-down menu you get when right-clicking a profile.